### DISCUSSION TITLE
Xcode Cloud

### DISCUSSION THREAD

**Discussion by nbasham** [pointfreeco/swift-snapshot-testing#740] [General]
Fantastic addition, thank you! My snapshots (SwiftUI previews - again, great job!) work locally but fail on Xcode Cloud. There has been much discussion on this going back over a year. Is there a resolution/solution to this? I've tried a couple of the suggestions in the discussion without success. Adding a note about this on the project read me would be helpful. Thank you!

**Comment by alexanderkhitev** [comment]
Hey guys! Any news/updates? Thanks!

**Comment by JuanColilla** [comment]
Can't have it working neither 😔

**Comment by fespinoza** [comment]
I made a guide on how to make this work in https://youtu.be/KPJL8vN3lAg
